### PR TITLE
Fix various storybook little issues

### PR DESCRIPTION
**Pull request by NathanVss**
## Purpose

- [x] allow storybook to use absolute paths
- [x] fix Storybook fonts loading

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   to screen reader users
 - Improve sale tunnel accessibility, especially when using a keyboard
   or screen reader
+- Fix storybook absolute paths
+- Fix storybook fonts loading
```
Comment (jbpenrath) [inline CHANGELOG.md:26]: As this concerns only configuration, we could remove these entries from changelog.

File: src/frontend/webpack.config.js [hunk 2]
```diff
@@ -63,6 +64,7 @@ module.exports = (env) => {
     resolve: {
       // Add '.ts' and '.tsx' as resolvable extensions.
       extensions: ['.ts', '.tsx', '.js', '.json'],
+      plugins: [new TsconfigPathsPlugin({})],
       modules: [
         path.resolve(__dirname, 'js'),
         __dirname,
```
Comment (jbpenrath) [inline src/frontend/webpack.config.js:70]: WIth the use of `TsconfigPathsPlugin`, we could remove `js` and `__dirname` from modules array.
```suggestion
      modules: [
```

File: CHANGELOG.md (hunk 1 above)
Comment (NathanVss) [reply CHANGELOG.md:26]: But the CI will not pass ? :/

File: src/frontend/package.json [hunk 3]
```diff
@@ -119,6 +119,7 @@
     "sass": "1.55.0",
     "source-map-loader": "4.0.0",
     "typescript": "4.8.4",
+    "tsconfig-paths-webpack-plugin": "4.0.0",
```
[the remark this answers is not in the document]
Comment (NathanVss) [reply src/frontend/package.json:122]: I forget this every-time 😅

**Review by jbpenrath — APPROVED** [review]
LGTM ✅

**Pull request merged**